src/EnumSet.php

```diff
@@ -24,9 +23,9 @@ class EnumSet implements Iterator, Countable
 
     /**
      * BitSet of all attached enumerations
-     * @var int
+     * @var string
      */
-    private $bitset = 0;
+    private $bitset = "";
```

Comment:
This variable don't need to be initialized as it will be initialized with a different value on constructor every time so the compiler don't need to reserve a string pointer and length information.